internal/cmd/config.go

```diff
@@ -747,18 +762,33 @@ traefik:
 	v.SetDefault("cluster::posthook::autoscaler::enabled", true)
 
 	// v.SetDefault("cluster::disasterRecovery::enabled", true)
-	v.SetDefault("cluster::disasterRecovery::namespace", "")
+	v.SetDefault("cluster::disasterRecovery::namespace", "pipeline-system")
 	v.SetDefault("cluster::disasterRecovery::ark::syncEnabled", true)
 	v.SetDefault("cluster::disasterRecovery::ark::bucketSyncInterval", "10m")
 	v.SetDefault("cluster::disasterRecovery::ark::restoreSyncInterval", "20s")
 	v.SetDefault("cluster::disasterRecovery::ark::backupSyncInterval", "20s")
 	v.SetDefault("cluster::disasterRecovery::ark::restoreWaitTimeout", "5m")
-	v.SetDefault("cluster::disasterRecovery::charts::ark::chart", "banzaicloud-stable/ark")
-	v.SetDefault("cluster::disasterRecovery::charts::ark::version", "1.2.3")
+	v.SetDefault("cluster::disasterRecovery::charts::ark::chart", "banzaicloud-stable/velero")
+	v.SetDefault("cluster::disasterRecovery::charts::ark::version", "2.13.3-bnz1")
```

Comment:
These version numbers kill me. We have `bnz`, `bzc` and god knows what else. It should either be something like `2.13.3-bc.1` or `2.13.3-banzaicloud.1` or just `2.13.3.1`.

Please come up with a better versioning strategy.